src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/ActiveHealthCheckOptions.cs

```diff
@@ -8,62 +8,56 @@ namespace Microsoft.ReverseProxy.Abstractions
     /// <summary>
     /// Active health check options.
     /// </summary>
-    public sealed class ActiveHealthCheckOptions
+    public sealed record ActiveHealthCheckOptions : IEquatable<ActiveHealthCheckOptions>
```

Comment:
I don't think we need to manually derive from `IEquatable`, that should `record` do automatically.
```suggestion
    public sealed record ActiveHealthCheckOptions
```